src/odemis/gui/cont/tabs.py

```diff
@@ -1706,6 +1706,101 @@ def get_display_priority(cls, main_data):
             return None
 
 
+class FastEMOverviewTab(Tab):
+    def __init__(self, name, button, panel, main_frame, main_data):
+
+        # During creation, the following controllers are created:
+        #
+        # ViewPortController
+        #   Processes given viewport.
+        #
+        # StreamController
+        #   Manages the single beam stream.
+        #
+        # Acquisition Controller
+        #   Takes care of the acquisition and acquisition selection buttons.
+
+        tab_data = guimod.FastEMOverviewGUIData(main_data)
+
+        super(FastEMOverviewTab, self).__init__(name, button, panel, main_frame, tab_data)
+        self.set_label("OVERVIEW")
+
+        # View Controller
+        vp = panel.vp_fastem_overview
+        assert(isinstance(vp, FastEMOverviewViewport))
+        vpv = collections.OrderedDict([
+            (vp,
+             {"name": "Overview",
+              "stage": main_data.stage,
+              "cls": guimod.StreamView,
+              "stream_classes": EMStream,
+              "zPos": self.tab_data_model.zPos,
+              }),
+        ])
+        self.view_controller = viewcont.ViewPortController(tab_data, panel, vpv)
+
+        # Single-beam SEM stream
+        vanames = ("resolution", "dwellTime", "horizontalFoV", "pixelSize")
+        main_data.hw_settings_config["e-beam"]["pixelSize"]["control_type"] = odemis.gui.CONTROL_READONLY
```

Comment:
conf.data already has way to override it per type of microscope, see `HW_SETTINGS_CONFIG_PER_ROLE`. If it should be just on this tab, you should make a copy of the dict, not change it on main.

Also, I wonder if we really want to show the pixel size? We never do that, partly because it's not the pixel size that the user expects: it's the pixel size *if scale == 1*.

Reply:
It was requested, but we were not aware that it's the pixel size if scale==1. I guess it's more confusing than helpful. It's definitely not essential, but I'll double check with Job if he still wants to have it.